tests/test_ble_interface_advanced.py

```
@@ -473,16 +476,18 @@ class BaseMockBleakClient:
         """Shared BLE client stub behavior for stress-test doubles."""
 
         def __init__(
-            self, address: str = "00:11:22:33:44:55", is_connected_result: bool = True
-        ):
+            self,
+            address: str = "00:11:22:33:44:55",
+            is_connected_result: bool = True,
```

Ruff (0.15.4)

[error] Boolean-typed positional argument in function definition

(FBT001)